src/Kdyby/Redis/ExclusiveLock.php

```diff
@@ -216,7 +224,7 @@ public function getLockTimeout($key)
 	 */
 	protected function formatLock($key)
 	{
-		return $key . ':lock';
+		return $this->keyPrefix . $key . ':lock';
```

Comment:
Is it really neccesary to prefix the lock keys? This would probably result in something like `instance1_instance1_:nette:cache:43243243242342432:lock` which isn't adding much value IMHO.